projects/core/src/cart/utils/utils.ts

```diff
@@ -1,9 +1,13 @@
 import { Cart } from '../../model/cart.model';
 import { OCC_USER_ID_ANONYMOUS } from '../../occ/utils/occ-constants';
 
-export function getCartIdByUserId(cart: Cart, userId: string) {
+export function getCartIdByUserId(cart: Cart, userId: string): string {
   if (userId === OCC_USER_ID_ANONYMOUS) {
     return cart.guid;
   }
   return cart.code;
 }
+
+export function isFreshCartId(cartId: string): boolean {
```

Comment:
It would be nice to document this function and describe:
- that it's not related to ids stored in backend, it's purely frontend
- that we need this temporary random IDs to observe (in components) the loading state of a cart that is being created (until we know it's real ID)
- we remove the temporary id as soon as we know the real ID
- why we need the random part - because 2 carts may be requested to be created at the same time (i.e. the active cart and the wish list) and we need to distinguish the loading state of both until they are created by the backend